client/src/pages/Prospecting/Prospecting.js

```diff
@@ -541,62 +563,156 @@ const Prospecting = () => {
           <ProspectingSummary period={period} summaryData={summaryData} />
         ) : (
           <>
-            <CustomTable
-              tableData={{
-                columns: columnShows,
-                data: detailedActivationData.map((item) => ({
-                  ...item,
-                  "account.name": (
-                    <Link
-                      href={`${instanceUrl}/${item.account?.id}`}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      {item.account?.name || "N/A"}
-                    </Link>
-                  ),
-                  "opportunity.name": item.opportunity ? (
-                    <Link
-                      href={`${instanceUrl}/${item.opportunity.id}`}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      {item.opportunity.name || "N/A"}
-                    </Link>
-                  ) : (
-                    "N/A"
-                  ),
-                })),
-                selectedIds: new Set(),
-                availableColumns: tableColumns,
-              }}
-              paginationConfig={{
-                type: "server-side",
-                totalItems: totalItems,
-                page: page,
-                rowsPerPage: rowsPerPage,
-                onPageChange: handlePageChange,
-                onRowsPerPageChange: handleRowsPerPageChange,
-              }}
-              onRowClick={handleRowClick}
-              onColumnsChange={handleColumnsChange}
-              isLoading={tableLoading}
-              onSearch={handleSearch}
-            />
-
+            <Box sx={{ flexGrow: 1, marginTop: 5 }}>
```

Comment:
This seems like it can go in its own component

Reply:
Possible or advisable to put all of the new view into an AccountDetail component?

Reply:
Ok, got it